.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "tpmlib"]
+	path = tpmlib
+	url = https://github.com/3mdeb/tpmlib.git
```

Comment:
Shouldn't it point at Trenchboot's tpmlib?

Reply:
Done